rand_core/src/lib.rs

```diff
@@ -314,11 +314,7 @@ pub trait SeedableRng: Sized {
             let xorshifted = (((state >> 18) ^ state) >> 27) as u32;
             let rot = (state >> 59) as u32;
             let x = xorshifted.rotate_right(rot).to_le();
```

Comment:
Can we move this `.to_le()` into `.to_le_bytes()`?

Reply:
Done. Good call.